content/id/docs/concepts/configuration/manage-compute-resources-container.md

```diff
@@ -0,0 +1,632 @@
+* `spec.containers[].resources.requests.memory`
+* `spec.containers[].resources.requests.hugepages-<size>`
+
+Walaupun `requests` dan `limits` hanya dapat diperinci pada Container individual, akan
+lebih mudah untuk membahas tentang request dan limit sumber daya dari Pod. Sebuah
+_request/limit sumber daya Pod_ untuk jenis sumber daya tertentu adalah jumlah dari
+request/limit sumber daya pada jenis tersebut untuk semua Container di dalam Pod tersebut.
+
+## Arti dari CPU
+
+Limit dan request untuk sumber daya CPU diukur dalam satuan _cpu_.
+Satu cpu, dalam Kubernetes, adalah sama dengan:
+
+- 1 vCPU AWS
+- 1 Core GCP
+- 1 vCore Azure
+- 1 vCPU IBM
+- 1 *Hyperthread* pada sebuah prosesor Intel _bare-metal_ dengan Hyperthreading
+
+Request dalam bentuk pecahan diizinkan. Sebuah Container dengan
+`spec.containers[].resources.requests.cpu` bernilai `0.5` dijamin mendapat
+setengah CPU dibandingkan dengan yang meminta 1 CPU. Ekspresi nilai  `0.1` ekuivalen
+dengan ekspresi bilai `100m`, yang dapat dibaca sebagai "seratus milicpu". Beberapa
+orang juga membacanya dengan "seratus milicore", dan keduanya ini dimengerti sebagai
+hal yang sama, Sebuah request dengan angka di belakang koma, seperti `0.1` dikonversi
+menjadi `100m` oleh API, dan presisi yang lebih kecil lagi dari `1m` tidak dibolehkan.
+Untuk alasan ini, bentuk `100m` mungkin lebih disukai.
+
+CPU juga selalu diminta dalam jumlah yang mutlak, tidak sebagai jumlah yang relatif;
+0.1 adalah jumlah CPU yang sama pada sebuah mesin _single-core_, _dual-core_, atau
+_48-core_.
+
+## Arti dari Memori
+
+Limit dan request untuk `memory` diukur dalam satuan _bytes_. Kamu dapat mengekspresikan
+memori sebagai _plain integer_ atau sebagai sebuah _fixed-point integer_ menggunakan
+satu dari sufiks-sufiks berikut: E, P, T, G, M, K. Kamu juga dapat menggunakan bentuk
+pangkat dua ekuivalennya: Ei, Pi, Ti, Gi, Mi, Ki.
+Sebagai contoh, nilai-nilai berikut kurang lebih sama:
+
+```shell
+128974848, 129e6, 129M, 123Mi
+```
+
+Berikut sebuah contoh.
+Pod berikut memiliki dua Container. Setiap Container memiliki request 0.25 cpu dan
+64MiB (2<sup>26</sup> bytes) memori. Setiap Container memiliki limit 0.5 cpu dan
+128MiB memori. Kamu dapat berkata bahwa Pod tersebut memiliki request 0.5 cpu dan
+128MiB memori, dan memiliki limit 1 cpu dan 265MiB memori.
+
+```yaml
+apiVersion: v1
+kind: Pod
+metadata:
+  name: frontend
+spec:
+  containers:
+  - name: db
+    image: mysql
+    env:
+    - name: MYSQL_ROOT_PASSWORD
+      value: "password"
+    resources:
+      requests:
+        memory: "64Mi"
+        cpu: "250m"
+      limits:
+        memory: "128Mi"
+        cpu: "500m"
+  - name: wp
+    image: wordpress
+    resources:
+      requests:
+        memory: "64Mi"
+        cpu: "250m"
+      limits:
+        memory: "128Mi"
+        cpu: "500m"
+```
+
+## Bagaimana Pod-Pod dengan request sumber daya dijadwalkan
+
+Saat kamu membuat sebuah Pod, Kubernetes scheduler akan memilih sebuah Node
```

Comment:
We might translate "scheduler" to "penjadwal" or, italicize it.

But, it is the name of a Kubernetes component:
https://kubernetes.io/docs/concepts/scheduling/kube-scheduler/

Do you guys think we should translate it, italicize it, or just leave it as is? @danninov @girikuncoro @irvifa @wahyuoi @qbl 
I think we can leave it as is for our translations to avoid confusion to the readers?

Reply:
agree with @phanama
leave it as is when it's Kubernetes component